npm/test/basics.js

```diff
@@ -59,38 +59,6 @@ namespace Test {
   assert(result.result === "Zero");
 });
 
-test("one syntax error", async () => {
```

Comment:
Are we not adding language service tests to cover these now? Looks like we're just deleting tests for functionality we are still keeping (via a different interface) here.

Reply:
All the functionality is covered elsewhere.

- `one syntax error`, `type error` - these validate that errors are reported. That's covered by the new tests I added in #426. (I don't see the need to have distinct tests for syntax and type errors.)

- `worker check` - validates error reporting in a web worker, also covered by #426

- `completions include CNOT` - completion behavior is covered by the [language service unit tests](https://github.com/microsoft/qsharp/blob/main/language_service/src/completion/tests.rs). I don't think it's that interesting to test from the JavaScript side since we're literally just wrapping the wasm object in a JS object and calling the same `getCompletions()` method.

- `error with newlines` - This is exactly the same as the wasm unit test [here](https://github.com/microsoft/qsharp/blob/6fcc83aaddb5cef382cdf7ad70cd635c7f1302c7/wasm/src/lib.rs#L212), it's just validating that the help text is formatted correctly. The npm layer doesn't add any new logic that needs to be validated.